stimuli-ci-generation/packages.cloudbuild.yaml

```diff
@@ -0,0 +1,81 @@
+# This build fetches R packages used on Kaggle and updates
+# the `packages_users` file on GitHub.
+
+# In order for this build to have write access to the repo, a deploy key
+# has been created and added to the repo.
+# - doc: https://cloud.google.com/cloud-build/docs/access-private-github-repos
+# - deploy key: https://github.com/Kaggle/docker-rcran/settings/keys
+
+steps:
+# Get the rsa key file from Secret Manager.
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: 'bash'
+  args: [ '-c', 'gcloud secrets versions access latest --secret=rcran-id-rsa > /root/.ssh/id_github' ]
+  volumes:
+  - name: 'ssh'
+    path: /root/.ssh
+
+# Set up git with key and domain.
+- name: 'gcr.io/cloud-builders/git'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    chmod 600 /root/.ssh/id_github
+    cat <<EOF >/root/.ssh/config
+    Hostname github.com
+    IdentityFile /root/.ssh/id_github
+    EOF
+    ssh-keyscan -t rsa github.com > /root/.ssh/known_hosts
+    git config --global user.email "robot@kaggle.com"
+  volumes:
+  - name: 'ssh'
+    path: /root/.ssh
```

Comment:
Is this here so we could work with Kaggle?

Reply:
I think the original author used a Kaggle project as a starting point/reference. We don't want to use Kaggle at all.